### ISSUE TITLE
HDMI output possible?

### ISSUE THREAD

**Issue by TychoWerner** [schoolpost/CinePI#20]
Hi, I would like to use this in live production, is it possible to have a clean feed from the HDMI?

**Comment by schoolpost** [comment]
Yes, you can make the preview image go out through HDMI. However, as of the current state; I have not been able to do simultaneous LCD preview + HDMI output ( only 1 or other. ) It should be possible but needs more development work make it possible.

**Comment by TychoWerner** [comment]
Ah okay, that is fine for my use case as I usd external monitors anyway

**Comment by lsdave** [comment]
Being able to control the camera using CEC over HDMI like how blackmagic does would be cool as well.

**Comment by PeteHaughie** [comment]
@TychoWerner you could try a HDMI splitter and feed it from the 5v GPIO pin.
